raiden/ui/checks.py

```diff
@@ -152,21 +152,6 @@ def check_ethereum_network_id(given_network_id: ChainID, web3: Web3) -> None:
         sys.exit(1)
 
 
-def check_raiden_environment(network_id: ChainID, environment_type: Environment) -> None:
```

Comment:
Why do you remove this check?

Reply:
Rakan told me he thinks --develop on mainnet is useful because it allows easier debugging.

Reply:
For features that are disabled on mainnet (e. g. token registration), we have explicit checks for network type rather than environment. So allowing `--develop` on mainnet should be fine, but I would still check for it and print a warning if it is chosen.

Reply:
That's a good idea.  I'll do that.